app/src/androidTest/java/org/mozilla/focus/activity/TestHelper.java

```diff
@@ -88,7 +88,7 @@
             .resourceId("org.mozilla.focus.debug:id/search_hint")
             .clickable(true));
     public static UiObject webView = mDevice.findObject(new UiSelector()
-            .className("android.webkit.Webview")
+            .className("android.webkit.WebView")
```

Comment:
major fail on my part :(

Reply:
Hehe, yeah, I noticed this too when using assertTrue(). ScreengrabTest shouldn't rely on this anymore - but a bunch of others.